src/core_plugins/console/public/src/autocomplete.js

```diff
@@ -9,11 +9,11 @@ import { populateContext } from './autocomplete/engine';
 import { URL_PATH_END_MARKER } from './autocomplete/url_pattern_matcher';
 import _ from 'lodash';
 import ace from 'ace';
-import 'ace/ext-language_tools';
+import 'brace/ext/language_tools';
```

Comment:
IIRC `brace` and `ace` are not the same module. Shouldn't we be using one or the other?

Reply:
This used to work because everything was copied into webpackShims under ace.  brace calls itself ace.  it's all confusing, but what is there works.